packages/flutter/lib/src/semantics/semantics_service.dart

```diff
@@ -57,6 +66,25 @@ abstract final class SemanticsService {
     await SystemChannels.accessibility.send(event.toMap());
   }
 
+
+  /// Sends a semantic announcement for a particular view.
+  ///
+  /// {@macro flutter.semantics.service.announce}
+  static Future<void> sendAnnouncement(
+    int viewId,
```

Comment:
Feels weird about forcing people to give an id, even for non-multiview case

Can we choose one of the following?

1. use kImplicitViewId if not provided

2. let SemanticsService to be a per View object instead of a global singleton, and use SemanticsService.of(context).annonce() instead, or View.of(context).semanticsService.announce

I slightly more leaning toward 2 since i imagine we may have more method added into this class in the future.

Reply:
> Do you mean that we should revert to the previous iteration of this PR where we deprecated the old non-multi-view method in favor of the new multi-view method? 

Yup that's right, but I'll defer to @chunhtai :)